nix/shell.nix

```diff
@@ -20,6 +20,8 @@ in
       wayland
       xorg.libxcb
       vulkan-loader
+      rustc
+      cargo
```

Comment:
There we provide rustc and cargo to the shell. In case of flake these will be versions from crane. Otherwise these will come from global `<nixpkgs>`.